assets/components/molecules/card-slider/card-slider.js

```diff
@@ -0,0 +1,46 @@
+/* globals $ */
+
+import Flickity from 'flickity';
+import Imagesloaded from 'imagesloaded';
+
+export default () => {
+  if ($('.card-slider').length > 0) {
+    const isMobile = $(window).width() < 767;
+
+    Imagesloaded('.card-slider', () => {
+      const flkty = new Flickity('.card-slider', {
+        cellAlign: 'left',
+        setGallerySize: true,
+        pageDots: isMobile,
+        prevNextButtons: false,
+        contain: true,
+        groupCells: false,
+      });
+
+      $('.card-slider-cell').css('height', '100%');
```

Comment:
Cannot be achieved with CSS?

Reply:
No and it's a JS widget who doesn't without it anyway.